pkgs/applications/office/planner/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchurl, meson, ninja, cmake, pkg-config,
+  desktop-file-utils, shared-mime-info, makeWrapper,
+  glib, gtk3, libxml2, libxslt, libgda, gsettings-desktop-schemas }:
+
+stdenv.mkDerivation rec {
+  pname = "planner";
+  version = "0.14.91";
+
+  src = fetchurl {
+    url = "https://gitlab.gnome.org/World/${pname}/-/archive/${version}/${pname}-${version}.tar.bz2";
+    sha256 = "1hd2adm6sg0dqwv32ykw4xv3q7x4hhjckxcx8p55amvvsy3jacy5";
```

Comment:
Use SRI hash.

Reply:
SRI hash? How do I get it?

Reply:
`nix hash to-sri sha256:<old format here>`